lib/connectors/connector_status.rb

```diff
@@ -0,0 +1,25 @@
+#
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. Licensed under the Elastic License;
+# you may not use this file except in compliance with the Elastic License.
+#
+
+# frozen_string_literal: true
+
+module Connectors
+  class ConnectorStatus
```

Comment:
nit - with `module Connectors`, the class name of `ConnectorStatus` seems redundant. Could it be `Connectors:Status` instead of `Connectors::ConnectorStatus`?

EDIT: ah, I see there's also a `sync_status` in the same dir. Ok, that makes more sense.